File: fabric/src/main/java/fudge/hardcraft/fabric/HardCraftFabric.java
package fudge.hardcraft.fabric;

import fudge.hardcraft.HardCraft;
import net.fabricmc.api.ModInitializer;

public class HardCraftFabric implements ModInitializer {

    @Override
    public void onInitialize() {
        HardCraft.initialize();
    }


}

File: fabric/src/main/kotlin/io/github/natanfudge/hardcraft/HardCraftFabric.kt
package io.github.natanfudge.hardcraft

import net.fabricmc.api.ModInitializer
import io.github.natanfudge.hardcraft.HardCraft
import net.fabricmc.api.ClientModInitializer
import net.minecraft.util.profiling.jfr.event.WorldLoadFinishedEvent

object HardCraftFabric: ModInitializer {
    override fun onInitialize() {
        HardCraft.init()

    }

}

object HardCraftFabricClient: ClientModInitializer {
    override fun onInitializeClient() {
        HardCraftClient.init()
    }
}
